app/src/main/cpp/skyline/services/am/controller/ICommonStateGetter.cpp

```diff
@@ -71,4 +71,9 @@ namespace skyline::service::am {
         }
         return {};
     }
+
+    // Stub implementation of SetCpuBoostMode
+    Result ICommonStateGetter::SetCpuBoostMode(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        return {};
```

Comment:
You should check for the value being valid here, probably write a `CpuBoostMode` enum then create `constexpr Result InvalidParameters(128, 506)` in the header under the `result` namespace and return it depending on if the value is valid.